[BUG] 前端构建报错：Module not found: Error: Can't resolve './JSEncrypt' in '/frontend/node_modules/jsencrypt/lib'

**问题描述**
 前端构建报错：Module not found: Error: Can't resolve './JSEncrypt' in '/frontend/node_modules/jsencrypt/lib'

**重现方法**
列出如何重现的方法或操作步骤 
1. 转到 '....'
2. 点击 '....'
3. 错误现象 '....'

**预期现象**
简明扼要地描述预期的现象是什么

**截屏**
请提供截屏来解释你的问题，当然这也能更好地帮助我们理解问题。

**请提供以下信息**

 - [x] bk-nodeman   版本 (发布版本号 或 git tag)： <!-- `示例： V3.1.32-ce 或者 git sha. 请不要使用 "最新版本" 或 "当前版本"等无法准确定位代码版本的语句描述`  -->
 - [ ] 蓝鲸PaaS   版本：<!-- `<示例：PaaS 3.0.58、PaaSAgent 3.0.9`  -->
 - [ ] bk-nodeman 异常日志：

--- a/dev_log/2.2.30/crayon_202210251609.yaml
+++ b/dev_log/2.2.30/crayon_202210251609.yaml
@@ -1,0 +1,3 @@
+---
+bugfix:
+  - "前端构建报错：Module not found: Error: Can't resolve './JSEncrypt' in '/frontend/node_modules/jsencrypt/lib' (fixed #1187)"

--- a/frontend/package.json
+++ b/frontend/package.json
@@ -60,7 +60,7 @@
     "bk-magic-vue": "2.4.9",
     "cookie": "~0.4.0",
     "github-markdown-css": "^4.0.0",
-    "jsencrypt": "^3.2.1",
+    "jsencrypt": "3.2.1",
     "jsonp": "~0.2.1",
     "query-string": "~6.5.0",
     "resize-detector": "^0.2.0",

